server/controllers/backup.js

```diff
@@ -46,9 +42,7 @@ module.exports = exports = async (req, res, next) => {
 		});
 	}
 	catch(error) {
-		log.error(`${MODULE_ID}`, {
-			error: error.stack
-		});
+		log.error({error});
```

Comment:
I think `MODULE_ID` (or some other distinct “thing”) would be very helpful here personally, for knowing what logged this